spec/models/transformation_mapping_item_spec.rb

```diff
@@ -100,4 +106,82 @@
       end
     end
   end
+
+  # ---------------------------------------------------------------------------
+  # Network Validation
+  # ---------------------------------------------------------------------------
+  context "Network validation" do
+    let(:ems_vmware) { FactoryBot.create(:ems_vmware) }
```

Comment:
This is a duplicate https://github.com/ManageIQ/manageiq/blob/2ae1efbb42de4776e8c4a10826454ffb7fc083fa/spec/models/transformation_mapping_item_spec.rb#L2

Reply:
Done.